<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trip</class>
 <widget class="QDialog" name="trip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="0" column="1">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QComboBox" name="dfs_comboBox"/>
         </item>
         <item>
          <widget class="QComboBox" name="bfs_comboBox"/>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="dfs_pushButton">
           <property name="text">
            <string>DFS </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bfs_pushButton">
           <property name="text">
            <string>BFS</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Starting at: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Starting at:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="mst_button">
         <property name="text">
          <string>MST</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTreeWidget" name="teamsTreeWidget">
      <column>
       <property name="text">
        <string>Available Teams</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="order_radioButton">
         <property name="text">
          <string>Keep Order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="done_pushButton">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clear_pushButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="displayButton">
      <property name="text">
       <string>Display</string>
      </property>
     </widget>
     <widget class="QListWidget" name="teamsList"/>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">background-color:#DBF3FA</string>
         </property>
         <property name="text">
          <string>Distance: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total_distance_label">
         <property name="styleSheet">
          <string notr="true">background-color:#DBF3FA</string>
         </property>
         <property name="text">
          <string>0.00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Yay buy stuff</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
